src/python/setup.py.in

```diff
@@ -41,8 +41,9 @@ for module in cython_modules:
         libraries=libraries,
         library_dirs=library_dirs,
         include_dirs=include_dirs,
-        runtime_library_dirs=runtime_library_dirs,
-        cython_directives = {'language_level': str(sys.version_info[0])},))
+        runtime_library_dirs=runtime_library_dirs,)
+    extension.cython_directives = {'language_level': str(sys.version_info[0])}
```

Comment:
Was the previous syntax wrong?

Reply:
I tried to fix language level warning but it failed.
```
FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release!
```
I can rollback this part.

Reply:
If the change didn't help, then yes, I'd rather keep the simpler version from before.

Reply:
Ok, I rolled back the changes.